composer.json

```diff
@@ -32,7 +32,7 @@
   "prefer-stable": true,
   "require": {
     "oat-sa/generis": "15.24.0",
-    "oat-sa/tao-core": "52.2.1",
+    "oat-sa/tao-core": "52.2.1.2",
```

Comment:
As a heads up, we could have created the tag `52.2.2` directly, cause it did not exist. We only need to create a fourth segment in case there is already a next fix version after the one we want to backport.

Reply:
I've created version `52.2.1.1`, then found an issue and had to create other release `52.2.1.2`

Why didn't I go with `52.2.2`. Because this backport introduces a feature, so the release should include minor version updat, not patch. We already have 52.3.x, so I could not increment minor version. 

May be this is worse than just creating patch version for a feature update, though

Reply:
Got it :) 

I did not know about this backporting rule, for me, regardless if it was an issue or fix, we just need to produce a new fix version available. 

Well, this is not a blocker, but mostly a heads up for next time. It will also produce cleaner versions tags IMO.

Thanks @shaveko